frame/treasury/src/tests.rs

```diff
@@ -163,21 +165,21 @@ fn genesis_config_works() {
 #[test]
 fn spend_origin_permissioning_works() {
 	new_test_ext().execute_with(|| {
-		assert_noop!(Treasury::spend(Origin::signed(1), 1, 1), BadOrigin);
+		assert_noop!(Treasury::propose_spend(Origin::signed(1), 1, 1, 0, false), BadOrigin);
 		assert_noop!(
-			Treasury::spend(Origin::signed(10), 6, 1),
+			Treasury::propose_spend(Origin::signed(10), 6, 1, 0, false),
```

Comment:
Change the `false` to `true` and the 0 to 1.

Reply:
This is test was testing the `spend` function. We do not use this function so we can comment out the test.